apps/src/code-studio/components/progress/MiniView.jsx

```diff
@@ -8,70 +8,68 @@ import {hasGroups} from '@cdo/apps/code-studio/progressRedux';
 /**
  * The course progress dropdown you get when you click the arrow in the header.
  */
-class MiniView extends React.Component {
-  static propTypes = {
-    linesOfCodeText: PropTypes.string,
-    minimal: PropTypes.bool,
+function MiniView(props) {
```

Comment:
I didn't change anything in this file besides making it a functional component. I was going to try to use useEffect but decided against it.